packages/api-genesis/src/example_queries/keyNetworkInfoByronShelley.graphql

```diff
@@ -0,0 +1,29 @@
+query keyNetworkInfo {
+  genesis {
+    byron {
+      blockVersionData {
+        scriptVersion
+        slotDuration
+        maxBlockSize
+        maxHeaderSize
+        maxProposalSize
+        maxTxSize
+      }
+      startTime
+      protocolConsts {
```

Comment:
These values are not Protocol specific, so could be moved out to `genesis.protocolConsts` rather than `genesis.byron.protocolConsts`. I think `startTime` could be added to it too?

Reply:
Also, to address this here...
The files are a 1:1 model of each genesis file, so while there are equivalents, I've chosen to align with the core structures taking them as domain models. Trying to align the two here would just create indirection IMO